lesson17/js/script.js

```diff
@@ -1,117 +1,124 @@
-// // task 1
+// Homework ------------------------------------------------------------
 
-// let a = 1;
-// let b = 1;
-// let c = ++a;
-// let d = b++;
+// Task 1 ------------------------------------------------------------
 
-// console.log(a, b, c, d);
+const strTrue = 'true';
```

Comment:
Рекомендуется не сокращать названия. Собеседования -- исключение
```suggestion
const stringTrue = 'true';
```